src/components/TimePicker/__test__/timePicker.spec.js

```diff
@@ -82,14 +83,14 @@ describe('<TimePicker/>', () => {
         });
     });
     it('should call to focusHourInput when the modal is open', () => {
```

Comment:
As this test no longer checks this, we should change this description to something like "should focus the hour input when modal is open"